src/dbnode/storage/series/buffer.go

```diff
@@ -56,11 +57,17 @@ var (
 
 const (
 	bucketsCacheSize = 2
-	// TODO(juchan): This represents the maximum number of blocks we think we'll
```

Comment:
Is this TODO still relevant?

Reply:
Fair, we should still monitor to see performance after ColdWrites are enabled.